### PR TITLE
fix bug

### PR DESCRIPTION
**Pull request by dfface**
跳过 label_ids 和 logits 中的 0 值。否则会报错 KeyError 0。
m 即 label_ids[i][j] ，而不是 j 。

### CODE REVIEW

**Comment by tlk1997** [comment]
m为标记好的label，范围是1到label_map的长度，所以不会出现0，logits为预测，存在为0的错误情况，所以可以不修改使用

**Comment by dfface** [comment]
> m为标记好的label，范围是1到label_map的长度，所以不会出现0，logits为预测，存在为0的错误情况，所以可以不修改使用

但是后面的代码直接使用了 `temp_2.append(label_map[logits[i][j]])` ，在当 `logits[i][j]` 为0的时候，就会报错，因为 label_map 里面没有为 0 的键。

**Comment by tlk1997** [comment]
> 

logits这的确会，后续将进行修改，感谢您的纠错
